src/definitions_cxx.hpp

```diff
@@ -623,9 +623,10 @@ enum class ModFXType {
 	CHORUS,
 	PHASER,
 	CHORUS_STEREO,
-	GRAIN,
+	GRAIN, // Look below if you want to add another one
 };
 
+// Warning: Currently GRAIN can be disabled and kNumModFXTypes might need to be used - 1
```

Comment:
this seems like a footgun - maybe add a getNumModFx function that returns the current value based on whether grain is on or off?

Reply:
Sorry accidentially resolved. I'd agree but accessing runtimeFeatureSettings from definitions_cxx.hpp isn't trivial and I didn't have a better idea to get this done quickly :D If you have a better idea lets change it! Just removing is also not the best way since it's non obvious changing the list will result in the access chagned in global_effectable.

Reply:
yeah, just tried and it breaks the compilation. This seems good since it's hopefully temporary anyway